test/parallel/test-tls-inception.js

```diff
@@ -33,7 +32,7 @@ a = tls.createServer(options, function(socket) {
   dest.pipe(socket);
   socket.pipe(dest);
 
-  dest.on('close', function() {
+  dest.on('end', function() {
```

Comment:
There are two ways to EOF for TLS connection: TCP FIN packet, and TLS close_notify alert (i.o.w. shutdown).